src/python/create.py

```diff
@@ -185,19 +188,42 @@ def create_materials(parsed_materials, project, data, citations):
             # If the material had a successful GET request, add properties, identifiers,
             # and select attributes as written in the excel
             else:
+                if newProject.name != project.name:
+                    material.project = project
+                    material.url = None
```

Comment:
I recommend comments for here, so it is obvious for any future developer of why we are setting them to **None** and they are not puzzled by this

Reply:
I agree. What are you trying to achieve assigning to None? Are you making a copy ?
If yes, I suggest to explain or maybe to declare a function (ex: `makeLocalCopy( output_material, input_material)`)